doc/perception_pipeline/launch/detect_and_add_cylinder_collision_object_demo.launch

```diff
@@ -0,0 +1,56 @@
+<launch>
+  <param name ="/use_sim_time" value="true"/>
+
+  <!-- By default, we are not in debug mode -->
+  <arg name="debug" default="false" />
+
+  <!--
+  By default, hide joint_state_publisher's GUI
+
+  MoveIt!'s "demo" mode replaces the real robot driver with the joint_state_publisher.
+  The latter one maintains and publishes the current joint configuration of the simulated robot.
+  It also provides a GUI to move the simulated robot around "manually".
+  This corresponds to moving around the real robot without the use of MoveIt.
+  -->
+  <arg name="rviz_tutorial" default="false" />
+  <arg name="use_gui" default="false" />
+
+  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
+  <include file="$(find panda_moveit_config)/launch/planning_context.launch">
+    <arg name="load_robot_description" value="true"/>
+  </include>
+
+  <!-- If needed, broadcast static tf for robot root -->
+  <node pkg="tf" type="static_transform_publisher" name="to_temp_link" args="0 0.4 -0.6 0 0 0  temp_link panda_link0 100" />
+  <node pkg="tf" type="static_transform_publisher" name="to_panda_base" args="0 0 0 0 0.2 1.92 camera_rgb_optical_frame temp_link 100" />
+  
+  <!-- Play the rosbag that contains the pointcloud data -->
+  <node pkg="rosbag" type="play" name="point_clouds" args="--clock -k $(find moveit_tutorials)/doc/perception_pipeline/bags/2018-05-28-07-49-11.bag" />
```

Comment:
I just created a S3 account for hosting this. If you send me this and other large files I can give you permalinks so that users can download them.

Reply:
[link](https://drive.google.com/file/d/14ZkTNgsexioHkJX5nDN4xr1i3KDhDkn6/view?usp=sharing)